plugin/src/helpers/utils.ts

```diff
@@ -177,7 +179,7 @@ export const findModuleFromBase = ({ paths, candidates }): string | null => {
 
 export const isNextAuthInstalled = (): boolean => {
   try {
-    // eslint-disable-next-line import/no-unassigned-import, import/no-unresolved, node/no-missing-require
+    // eslint-disable-next-line n/global-require, import/no-unassigned-import, import/no-unresolved, n/no-missing-require
```

Comment:
A lot of these changes are related to moving from `eslint-plugin-node` to `eslint-plugin-n`. See https://github.com/netlify/eslint-config-node/releases/tag/v6.0.0